restli-client/src/main/java/com/linkedin/restli/client/BatchGetKVRequest.java

```diff
@@ -1,72 +0,0 @@
-/*
-   Copyright (c) 2012 LinkedIn Corp.
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-*/
-
-package com.linkedin.restli.client;
-
-
-import com.linkedin.data.schema.PathSpec;
-import com.linkedin.data.template.RecordTemplate;
-import com.linkedin.restli.client.response.BatchKVResponse;
-import com.linkedin.restli.common.ResourceMethod;
-import com.linkedin.restli.common.ResourceSpec;
-import com.linkedin.restli.internal.client.RestResponseDecoder;
-
-import java.net.HttpCookie;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
-
-/**
- * A request for reading a resource batch.
```

Comment:
Yes, you're right. I think we can skip this step unless we have a formal deprecation policy that we want to follow strictly, because it appears that the only reason why this and similar deleted classes are needed is if the deprecated builders were used to build requests. That is, these classes were for all intents and purposes deprecated.